dvc/commands/plots.py

```diff
@@ -74,51 +147,44 @@ def run(self):
             renderers_out = (
                 out if self.args.json else os.path.join(out, "static")
             )
-            renderers = match_renderers(
-                plots_data=plots_data,
+
+            renderers = match_defs_renderers(
+                pathlib=self.repo.fs.path,
```

Comment:
When invoking `show/diff` from different dir than `root` of the project we need to evaluate data source relative path (to the cwd) to relative path of data (which is relative  to `dvc.yaml` where plot is defined).

Reply:
I guess my question is why it's required as part of the renderer matching step? 
Is it possible to resolve that at collection time?

Reply:
Huh, good point, we shouldn't need that here if we resolved everything during collection time. This is because I keep the plots definitions unchanged (as they are in particular dvc.yaml), I should probably reconsider that.

Reply:
Should be fixed - I am not sure that new version is better as it modifies plots definitions depending on `cwd` but on the other hand it does not require us to resolve the paths in every place that we need to use plots.